lighthouse-core/report/html/report-styles.css

```diff
@@ -652,14 +652,16 @@
   border-bottom: none;
 }
 .lh-audit-group--metrics .lh-audit-group__summary {
+  margin-top: 0;
   margin-bottom: 0;
 }
 
 .lh-audit-group__summary {
   display: flex;
   justify-content: space-between;
   padding-right: var(--text-indent);
-  margin-bottom: var(--lh-audit-vpadding);
+  margin-top: calc(var(--section-padding) * 1.5);
+  margin-bottom: var(--section-padding);
```

Comment:
`--section-padding` is admittedly somewhat a pick out of a hat, but it mostly matches what's being done here, and it gets the slightly more crowded style for devtools (24/16px by default, 18/12px in devtools)

(plus @paulirish suggested using it for this :)